microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -33,30 +33,43 @@ The Microsoft Edge Add-ons API provides a set of REST endpoints for programmatic
 <!-- ====================================================================== -->
 ## Before you begin
 
-To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center.
+To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center, by creating API credentials.
 
 > [!NOTE]
-> The **Publish APIs** UI is not yet present at Partner Center.
+> The **Publish API** page is only present at Partner Center for participants of the private preview.
 
-1. Visit Microsoft Partner Center and login to the account from which you have already published an add-on.
+1. Visit Microsoft Partner Center and sign in to the account from which you have already published an add-on.
```

Comment:
note fyi: Changed login to sign in per Microsoft Writing Style Guide. https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/sign-in-sign-out